loader/CMakeLists.txt

```diff
@@ -95,6 +95,7 @@ else()
     if(CMAKE_SYSTEM_NAME MATCHES "FreeBSD")
         target_compile_definitions(loader_specific_options INTERFACE __BSD_VISIBLE=1)
     endif()
+    check_include_file("alloca.h" HAVE_ALLOCA_H)
 endif()
```

Comment:
We should use `target_compile_definitions` to apply defines into the code. As it currently is used, `HAVE_ALLOCA_H` would *always* be undefined when the loader is compiled.